--- a/programs/dthelp/dthelpview/Main.c	2020-01-06 04:53:03.000000000 +0100
+++ b/programs/dthelp/dthelpview/Main.c	2020-01-17 11:05:12.659999575 +0100
@@ -74,7 +74,7 @@
 /* Local Includes */
 
 #include <DtI/HelposI.h>
-#include <DtI/HelpP.h>
+#include <Dt/HelpP.h>
 #include <DtI/FileUtilsI.h>
 #include "Main.h"
 #include "UtilI.h"
--- a/programs/dthelp/dthelpview/Makefile.am	2020-01-06 04:53:03.000000000 +0100
+++ b/programs/dthelp/dthelpview/Makefile.am	2020-01-17 11:08:38.258883570 +0100
@@ -4,6 +4,6 @@
 
 dthelpview_CFLAGS = -D_BMS
 
-dthelpview_LDADD = $(LIBHELP) $(LIBSVC) $(LIBTT) $(XTOOLLIB) ${X_LIBS}
+dthelpview_LDADD = $(LIBHELP) $(LIBSVC) $(LIBTT) $(LIBXIN) $(XTOOLLIB) ${X_LIBS}
 
 dthelpview_SOURCES = Main.c ManPage.c Util.c
--- a/programs/dthelp/dthelpview/Util.c	2020-01-06 04:53:03.000000000 +0100
+++ b/programs/dthelp/dthelpview/Util.c	2020-01-17 11:05:24.836600099 +0100
@@ -58,7 +58,7 @@
 #include <Xm/Text.h>
 
 #include <Dt/Help.h>
-#include <DtI/HelpP.h>
+#include <Dt/HelpP.h>
 #include <Dt/HelpDialog.h>
 #include <Dt/HelpQuickD.h>

--- a/programs/dthelp/Makefile.am	2020-01-06 04:53:03.000000000 +0100
+++ b/programs/dthelp/Makefile.am	2020-01-17 12:00:15.482181425 +0100
@@ -1,3 +1,4 @@
 MAINTAINERCLEANFILES = Makefile.in
 
-SUBDIRS = parser dthelpview dthelpdemo dthelpgen dthelpprint
+#SUBDIRS = parser dthelpview dthelpdemo dthelpgen dthelpprint
+SUBDIRS = dthelpview
