--- a/configure.ac	2020-01-25 19:07:21.265289725 +0100
+++ b/configure.ac	2020-01-27 08:44:10.455457862 +0100
@@ -494,6 +494,23 @@
 
 programs/dsdm/Makefile
 
+programs/dthelp/Makefile
+programs/dthelp/dthelpview/Makefile
+programs/dthelp/dthelpgen/Makefile
+programs/dthelp/dthelpdemo/Makefile
+
+programs/dtinfo/Makefile
+programs/dtinfo/dtinfogen/Makefile
+programs/dtinfo/dtinfogen/mmdb/Makefile
+programs/dtinfo/dtinfogen/mmdb/src/Makefile
+programs/dtinfo/clients/Makefile
+programs/dtinfo/clients/dtinfo_start/Makefile
+programs/dtinfo/dtinfo/install/Makefile
+programs/dtinfo/dtinfo/install/lib/Makefile
+programs/dtinfo/dtinfo/install/lib/bitmaps/Makefile
+
+programs/nsgmls/Makefile
+
 programs/dtmail/Makefile
 programs/dtmail/dtmail/Makefile
 programs/dtmail/MotifApp/Makefile

