--- a/configure.ac	2020-01-06 04:53:03.000000000 +0100
+++ b/configure.ac	2020-01-18 22:13:31.264887940 +0100
@@ -494,6 +494,21 @@
 
 programs/dsdm/Makefile
 
+programs/dthelp/Makefile
+programs/dthelp/dthelpview/Makefile
+
+programs/dtinfo/Makefile
+programs/dtinfo/dtinfogen/Makefile
+programs/dtinfo/dtinfogen/mmdb/Makefile
+programs/dtinfo/dtinfogen/mmdb/src/Makefile
+programs/dtinfo/clients/Makefile
+programs/dtinfo/clients/dtinfo_start/Makefile
+programs/dtinfo/dtinfo/install/Makefile
+programs/dtinfo/dtinfo/install/lib/Makefile
+programs/dtinfo/dtinfo/install/lib/bitmaps/Makefile
+
+programs/nsgmls/Makefile
+
 programs/dtmail/Makefile
 programs/dtmail/dtmail/Makefile
 programs/dtmail/MotifApp/Makefile
--- a/programs/Makefile.am	2020-01-06 04:53:03.000000000 +0100
+++ b/programs/Makefile.am	2020-01-18 15:04:17.260678184 +0100
@@ -5,6 +5,6 @@
 	dtaction dtspcd dtscreen dtcm dtsearchpath palettes \
 	dtappintegrate dtprintegrate dtconfig dtcreate dtprintinfo fontaliases \
 	dtdspmsg dtimsstart dtpdm dtsr dtpdmd types \
-	tttypes util dtopen localized
+	tttypes util dtopen localized dthelp nsgmls
 
-# NOTYET = dthelp dtksh dtappbuilder dtdocbook dtinfo nsgmls ttsnoop 
+# NOTYET = dtksh dtappbuilder dtdocbook dtinfo ttsnoop 
