--- a/programs/Makefile.am	2020-01-06 04:53:03.000000000 +0100
+++ b/programs/Makefile.am	2020-01-17 11:44:10.850620890 +0100
@@ -5,6 +5,6 @@
 	dtaction dtspcd dtscreen dtcm dtsearchpath palettes \
 	dtappintegrate dtprintegrate dtconfig dtcreate dtprintinfo fontaliases \
 	dtdspmsg dtimsstart dtpdm dtsr dtpdmd types \
-	tttypes util dtopen localized
+	tttypes util dtopen localized dthelp
 
-# NOTYET = dthelp dtksh dtappbuilder dtdocbook dtinfo nsgmls ttsnoop 
+# NOTYET = dtksh dtappbuilder dtdocbook dtinfo nsgmls ttsnoop 
