--- a/configure.ac	2020-01-26 20:22:09.000000000 +0100
+++ b/configure.ac	2020-01-28 13:58:47.760143923 +0100
@@ -494,6 +494,13 @@
 
 programs/dsdm/Makefile
 
+programs/dthelp/Makefile
+programs/dthelp/dthelpview/Makefile
+programs/dthelp/dthelpgen/Makefile
+programs/dthelp/dthelpdemo/Makefile
+
+programs/nsgmls/Makefile
+
 programs/dtmail/Makefile
 programs/dtmail/dtmail/Makefile
 programs/dtmail/MotifApp/Makefile
--- a/programs/Makefile.am	2020-01-26 20:22:09.000000000 +0100
+++ b/programs/Makefile.am	2020-01-28 13:56:42.114156893 +0100
@@ -5,6 +5,6 @@
 	dtaction dtspcd dtscreen dtcm dtsearchpath palettes \
 	dtappintegrate dtprintegrate dtconfig dtcreate dtprintinfo fontaliases \
 	dtdspmsg dtimsstart dtpdm dtsr dtpdmd types \
-	tttypes util dtopen localized
+	tttypes util dtopen localized dthelp nsgmls
 
-# NOTYET = dthelp dtksh dtappbuilder dtdocbook dtinfo nsgmls ttsnoop 
+# NOTYET = dtksh dtappbuilder dtdocbook dtinfo ttsnoop 
