--- a/programs/dtmail/dtmail/Makefile.am	2020-01-16 11:09:46.943417096 +0100
+++ b/programs/dtmail/dtmail/Makefile.am	2020-01-16 11:37:22.242128423 +0100
@@ -72,6 +72,6 @@
 		 options_ui.C                    options_util.C
 
 install-exec-hook:
-	chown root $(DESTDIR)$(CDE_INSTALLATION_TOP)/bin/dtmail
-	chgrp mail $(DESTDIR)$(CDE_INSTALLATION_TOP)/bin/dtmail
-	chmod 2755 $(DESTDIR)$(CDE_INSTALLATION_TOP)/bin/dtmail
+	chown root $(DESTDIR)$(bindir)/dtmail
+	chgrp mail $(DESTDIR)$(bindir)/dtmail
+	chmod 2755 $(DESTDIR)$(bindir)/dtmail
--- a/programs/dtsearchpath/dtappg/Makefile.am	2020-01-16 11:09:47.370045415 +0100
+++ b/programs/dtsearchpath/dtappg/Makefile.am	2020-01-16 11:44:23.747774831 +0100
@@ -13,5 +13,5 @@
 dtappgather_SOURCES = Options.C dtappgather.C dtappgather.h Options.h
 
 install-exec-hook:
-	chown root $(DESTDIR)$(CDE_INSTALLATION_TOP)/bin/dtappgather
-	chmod 4755 $(DESTDIR)$(CDE_INSTALLATION_TOP)/bin/dtappgather
+	chown root $(DESTDIR)$(bindir)/dtappgather
+	chmod 4755 $(DESTDIR)$(bindir)/dtappgather
--- a/programs/dtsession/Makefile.am	2020-01-16 11:09:47.403375752 +0100
+++ b/programs/dtsession/Makefile.am	2020-01-16 11:46:27.095854972 +0100
@@ -119,6 +119,6 @@
 
 
 install-exec-hook:
-	chown root $(DESTDIR)$(CDE_INSTALLATION_TOP)/bin/dtsession
-	chmod 4755 $(DESTDIR)$(CDE_INSTALLATION_TOP)/bin/dtsession
+	chown root $(DESTDIR)$(bindir)/dtsession
+	chmod 4755 $(DESTDIR)$(bindir)/dtsession
 
--- a/programs/dtterm/Makefile.am	2020-01-16 11:09:47.546696203 +0100
+++ b/programs/dtterm/Makefile.am	2020-01-16 11:49:31.231497608 +0100
@@ -44,6 +44,6 @@
 	CPP="$(GENCPP)" $(SHELL) terminfoCreate < terminfoChecklist > $@
 
 install-exec-hook:
-	chown root $(DESTDIR)$(CDE_INSTALLATION_TOP)/bin/dtterm
-	chmod 4755 $(DESTDIR)$(CDE_INSTALLATION_TOP)/bin/dtterm
+	chown root $(DESTDIR)$(bindir)/dtterm
+	chmod 4755 $(DESTDIR)$(bindir)/dtterm
 
